e2etest/pkg/frr/container/container.go

```diff
@@ -365,3 +374,30 @@ func containerIsRunning(containerName string) error {
 
 	return nil
 }
+
+func SetupP2PPeer(frrImage string, node corev1.Node) (*FRR, error) {
+	c := Config{
+		Name:    fmt.Sprintf("unnumbered-p2p-%d", time.Now().Unix()),
```

Comment:
wdyt about naming the peer after the container it is peered with? 
```go
fmt.Sprintf("p2p-%s", containerName)
```

Reply:
done